### ISSUE TITLE
Import biomass: San Pablo Valdez 2015 - 2021

### ISSUE THREAD

**Issue by KW529** [felt0134/drought-net-tickets#182]

**Comment by KW529** [comment]
Mendy e-mail (04-26-2020):

"Hi Martin,
We would definitely like to include your site. Can you please confirm that you have submitted an estimate of aboveground net primary production (grams per meter squared)? Given your response in the survey last year, you indicated that you did not measure ANPP in g*m-2. Thus we were not including your site in the biomass analyses. Please let us know and we will be happy to include your site if it meets the criteria.
Best,
Mendy"

**Comment by KW529** [comment]
My follow-up e-mail (07-02-2020): 

"Hi Martin,

I hope all is well! I am following up on Mendy's e-mail inquiring about whether your site reported aboveground net primary production. I have re-pasted Mendy's original e-mail below: 

"We would definitely like to include your site. Can you please confirm that you have submitted an estimate of aboveground net primary production (grams per meter squared)? Given your response in the survey last year, you indicated that you did not measure ANPP in g*m-2. Thus we were not including your site in the biomass analyses. Please let us know and we will be happy to include your site if it meets the criteria."

Thanks so much for any clarification on this!

Best,
Kate"

**Comment by KW529** [comment]
Martin forwarded an e-mail to Mendy from 4-28-2020:

"Dear Melinda,
Below this mail you can find the replay of their mail on November 5 of 2019 when we explaing you how we evaluated the ANPP data from San Pablo de Valdes (Chubut, Argentina). As it stands on the protocol, we collected aboveground plant biomass at its peak, which could be considered a proxy to estimate ANPP. We hope that our data could be included in the analyses. Please let us know if you need clarifications

Kind regards,

Martin Bruschetti"

**Comment by KW529** [comment]
Data entered

**Comment by KW529** [comment]
Hi Martin,

I hope all is well! Thanks so much for sharing your 2020 data with us. I have a quick question about plot 5 from the 2020 biomass data. There appears to be two values for 'Total' biomass in plot 5. The 137.5 value looks correct, but I wanted to double check to make sure that I should discard the Total=370.

site | plot | subplot | taxa | mass | date
-- | -- | -- | -- | -- | --
San Pablo de Valdés | 5 | A | Total | 370 | 11/28/2020
San Pablo de Valdés | 5 | A | Graminoid | 52 | 11/28/2020
San Pablo de Valdés | 5 | A | Litter | 85.5 | 11/28/2020
San Pablo de Valdés | 5 | A | Total | 137.5 | 11/28/2020

Best, 
Kate

**Comment by KW529** [comment]
Oct 20, 2021: 

Hi Kate,

Attached I send you the corrected version of the 2020 data from San Pablo Valdes, Argentina.

Thanks

Best regards,

Martin Bruschetti

**Comment by KW529** [comment]
Feb 22, 2022:
hola melinda!!

I am writing to you because we found an inconsistency in the 2020 and
2021 data spreadsheets from San Pablo de Valdes, Argentina. Now, I am sending you the new corrected spreadsheets. It was just one plot (2) that was changed.

on the other hand, our doubt is if this year we also have to take samples from the plots or does the study end?

Thanks

best regards

Martin Bruschetti

UNMdP-IIMyC
